test/UnsafeTest/src_90/org/openj9/test/unsafe/MainTester.java

```diff
@@ -45,22 +45,28 @@
 			compileClass();
 		}
 		List<Object> result = new ArrayList<>();
+		result.add(new TestUnsafePutGetAddress(scenario));
```

Comment:
There are one java file deleted and 5 java file added. Changing the order makes this block seems adding 8 files and delete 5 files. Can we keep the previous order and add new files following the alphabet order?

Reply:
Thanks for the updates